openwebstart/src/main/java/com/openwebstart/os/mac/MacEntryFactory.java

```diff
@@ -25,29 +25,40 @@
 public class MacEntryFactory implements MenuAndDesktopEntriesFactory {
 
     @Override
-    public boolean supportsDesktopEntry() {
-        return false;
+    public boolean supportsDesktopEntry() 
+    {
+        return true;
     }
 
     @Override
-    public void updateDesktopEntry(final JNLPFile file) {
-        throw new RuntimeException("Operation not supported");
+    public void updateDesktopEntry(final JNLPFile file)
```

Comment:
You are using a complete different code format than everyone in the project. Can you please format your code in the default format of the project.